addons/l10n_pe/i18n/l10n_pe.pot

```diff
@@ -0,0 +1,9611 @@
+#: model:res.city,name:l10n_pe.city_pe_0808
+msgid "Espinar"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_230403
+msgid "Estique"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_230404
+msgid "Estique-Pampa"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_140103
+msgid "Eten"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_140104
+msgid "Eten Puerto"
+msgstr ""
+
+#. module: l10n_pe
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax__l10n_pe_edi_unece_category__e
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax_template__l10n_pe_edi_unece_category__e
+msgid "Exempt from tax"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_160103
+msgid "Fernando Lores"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_140201
+#: model:res.city,name:l10n_pe.city_pe_1402
+msgid "Ferreñafe"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_021304
+msgid "Fidel Olivas Escudero"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_170202
+msgid "Fitzcarrald"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_130103
+msgid "Florencia de Mora"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_010306
+msgid "Florida"
+msgstr ""
+
+#. module: l10n_pe
+#: model:ir.model.fields,help:l10n_pe.field_account_tax_template__l10n_pe_edi_unece_category
+msgid ""
+"Follow the UN/ECE 5305 standard from the United Nations Economic Commission "
+"for Europe for more information  "
+"http://www.unece.org/trade/untdid/d08a/tred/tred5305.htm"
+msgstr ""
+
+#. module: l10n_pe
+#: model:ir.model.fields,help:l10n_pe.field_account_tax__l10n_pe_edi_unece_category
+msgid ""
+"Follow the UN/ECE 5305 standard from the United Nations Economic Commission "
+"for Europe for more information "
+"http://www.unece.org/trade/untdid/d08a/tred/tred5305.htm"
+msgstr ""
+
+#. module: l10n_pe
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax__l10n_pe_edi_unece_category__g
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax_template__l10n_pe_edi_unece_category__g
+msgid "Free export item, tax not charged"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_200202
+msgid "Frias"
+msgstr ""
+
+#. module: l10n_pe
+#: model:account.tax.group,name:l10n_pe.tax_group_gra
+msgid "GRA"
+msgstr ""
+
+#. module: l10n_pe
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax__l10n_pe_edi_tax_code__9996
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax_template__l10n_pe_edi_tax_code__9996
+msgid "GRA - Free"
```

Comment:
@umiphos same comment than before.

Reply:
the POT file is not translated it's only a template file for translations, in the es.po file is the real spanish translation